bol/attachment_dao.php

```diff
@@ -194,6 +194,29 @@ public function findConversationsWithAttachmentFromConversationList($conversatio
 
         return $this->dbo->queryForColumnList($sql);
     }
+    
+    public function getAttachmentFilesForDelete()
+    {
+          $sql = "SELECT ow_mailbox_attachment.id as attachId, hash, fileName, ow_mailbox_attachment.messageId AS messageId, ow_mailbox_message.id
```

Comment:
Do not use hardcoded table name. You need use `$this->getTableName()` method of DAO